cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -77,6 +77,12 @@ spec:
   # enable the crash collector for ceph daemon crash collection
   crashCollector:
     disable: false
```

Comment:
After we settle on the review of this PR, how about if we cherry-pick this commit into #4959 and close this PR? Then we can test it all and merge together.